libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-audit-logs.component.html

```diff
@@ -15,12 +15,14 @@
 limitations under the License.
 -->
 <watt-card variant="solid" *transloco="let t; read: 'admin.userManagement.tabs.history'">
-  <div *ngIf="isLoading$ | push" class="spinner">
-    <watt-spinner />
-  </div>
+  @if (isLoading$ | push) {
+    <div class="spinner">
+      <watt-spinner />
+    </div>
+  }
 
-  <ng-container *ngIf="!(isLoading$ | push)">
-    <ng-container *ngIf="!(hasFailed$ | push)">
+  @if (!(isLoading$ | push)) {
```

Comment:
Suggestion: Combine these two in to one if?

Reply:
Should be fixed now.